packages/showcases/svelte/stories/components/indicators/VtmnRating/VtmnRating.stories.svelte

```diff
@@ -0,0 +1,198 @@
+<script>
+  import { Meta, Template, Story } from '@storybook/addon-svelte-csf';
+  import { VtmnRating } from '@vtmn/svelte';
+  import { parameters } from '@vtmn/showcase-core/csf/components/indicators/rating.csf';
+  const argTypes = {
+    name: {
+      type: { name: 'string', required: false },
+      description: 'Name used on the input radio',
+      defaultValue: 'rating',
+      control: {
+        type: 'text',
+      },
+    },
+    emphasis: {
+      type: { name: 'boolean', required: false },
+      description: 'The variant of the rating.',
+      defaultValue: false,
+      control: {
+        type: 'boolean',
+      },
+    },
+    size: {
+      type: { name: 'string', required: false },
+      description: 'Size of the rating.',
+      defaultValue: 'medium',
+      control: {
+        type: 'select',
+        options: ['small', 'medium'],
+      },
+    },
+    disabled: {
+      type: { name: 'boolean', required: false },
+      description: 'Whether the rating is disabled.',
+      defaultValue: false,
+      control: { type: 'boolean' },
+    },
+    readOnly: {
+      type: { name: 'boolean', required: false },
+      description: 'Whether the rating is read only.',
+      defaultValue: true,
+      control: { type: 'boolean' },
+    },
+    compact: {
+      type: { name: 'number', required: false },
+      description: 'How many stars to display.',
+      defaultValue: false,
+      control: { type: 'boolean' },
+    },
+    rating: {
+      type: { name: 'boolean', required: false },
+      description: 'How many stars are filled.',
+      defaultValue: 0,
+      control: { type: 'range', min: 0, max: 5, step: 0.25 },
+    },
+  };
+</script>
+
+<Meta
+  title="Components / Indicators / VtmnRating"
+  component={VtmnRating}
+  {argTypes}
+  {parameters}
+/>
+
+<Template let:args>
+  <VtmnRating {...args} />
+</Template>
+
+<Story name="Overview" />
+
+<Story name="Interactive">
+  <VtmnRating
+    class="rating-stories"
+    name="story-interfactive-1"
+    emphasis
+    size="medium"
+    rating={0}
+  />
+  <VtmnRating
+    class="rating-stories"
+    name="story-interfactive-2"
+    emphasis
+    size="small"
+    rating={0}
+  />
+  <VtmnRating
+    class="rating-stories"
+    name="story-interfactive-3"
+    emphasis
+    disabled
+    rating={0}
+  />
+</Story>
+
+<Story name="Read Only with text">
+  <VtmnRating class="rating-stories" readOnly="true" compact rating={1}>
+    <span slot="primary">4.1/5</span>
+  </VtmnRating>
+  <VtmnRating class="rating-stories" readOnly="true" compact rating={0.1}>
```

Comment:
I don't understand the `rating={0.1}`?

Reply:
Rating 0.1 can test this case : 
![image](https://user-images.githubusercontent.com/2856778/157913640-7133205b-dffd-4bc5-906a-f9b9ea407bb8.png)